modules/calcite/src/test/sql/subquery/scalar/test_scalar_subquery.test_ignore

```diff
@@ -102,29 +102,18 @@ SELECT a, (SELECT SUM(b) FROM test tsub WHERE test.a=tsub.a) FROM test
 12	21.000000
 13	22.000000
 
-query II
+statement error
```

Comment:
Why statement error. The request seems not failing

Reply:
Fixed, thanks